### PR TITLE
VIP code review fixes

### PR DESCRIPTION
**Pull request by SolomonSScott**

### CODE REVIEW

**Comment by SolomonSScott** [comment]
@reaktivstudios/reviewers this is ready for review.

File: knightlab-timeline.php [hunk 1]
```diff
@@ -16,255 +16,280 @@
...
+	$img          = KL_TIMELINE_URL . 'knight-diamond.png';
+	$container_id = 'add_timeline_form';
+	$title        = __( 'Insert Timeline', 'kl-timeline' );
 
-  //append the icon
-  $context .= "<a class='thickbox button' style='background: url({$img}) no-repeat 3px 3px; padding-left: 20px' title='{$title}' id='add_timeline'
-    href='#TB_inline?width=600&height=495&inlineId={$container_id}'> Add Timeline</a>";
+	// append the icon.
+	$context .= "<a class='thickbox button' style='background: url({$img}) no-repeat 3px 3px; padding-left: 20px' title='{$title}' id='add_timeline'
+	href='#TB_inline?width=600&height=495&inlineId={$container_id}'> Add Timeline</a>";
```
Comment (corywebb) [inline knightlab-timeline.php]: Needs i18n.

File: knightlab-timeline.php [hunk 2]
```diff
@@ -16,255 +16,280 @@
...
-				var maptype = jQuery('#timeline_maptype').val();
-			} catch(e) {
-				console.log(e);
-			}
+	function insertTimeline() {
+		var data_src = jQuery('#timeline_data_src').val();
+		var height = jQuery('#timeline_height').val();
+		var width = jQuery('#timeline_width').val();
+		var lang = jQuery('#timeline_lang').val();
+		var font = jQuery('#timeline_font').val();
```
Comment (corywebb) [inline knightlab-timeline.php]: This section should be done with a single `var`.

File: knightlab-timeline.php [hunk 3]
```diff
@@ -16,255 +16,280 @@
...
+									<option value="sv" data-lang="Swedish">Svenska</option>
+									<option value="zh-tw" data-lang="Taiwanese">Taiwanese</option>
+									<option value="tl" data-lang="Tagalog">Tagalog</option>
+									<option value="ta" data-lang="Tamil">தமிழ்</option>
+									<option value="te" data-lang="Telugu">తెలుగు</option>
+									<option value="tr" data-lang="Turkish">Türkçe</option>
+									<option value="uk" data-lang="Ukrainian">Українська</option>
+									<option value="ur" data-lang="Urdu">اُردُو</option>
+									<option value="fa" data-lang="Farsi">فارسی</option>
+									<option value="zh-cn" data-lang="Chinese">中文</option>
```
Comment (corywebb) [inline knightlab-timeline.php]: All the `<option>` tags need 1 more tab.

**Review by corywebb — CHANGES_REQUESTED** [review]
Just a couple of little things.

**Review by corywebb — APPROVED** [review]
LGTM

**Pull request merged**